<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgIOTriggerClass</class>
 <widget class="QDialog" name="DlgIOTriggerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgIOTrigger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>638</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_close">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 8px 16px 8px 36px; 
    font-size: 14px;
    qproperty-icon: url(:/ButtonScanner/image/exit.png);  
    qproperty-iconSize: 30px 30px;
}

QPushButton:hover {
    background-color: #ff6666;
}

QPushButton:pressed {
    background-color: #cc0000;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="gbox_IO">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_shoudongladai">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: red;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
}

QPushButton:pressed {
    border-color: #388E3C;
background-color: #4CAF50; /* 明显的绿色 */
color: #fff;
font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>手动拉袋</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_tuoji">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: red;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
}

QPushButton:pressed {
    border-color: #388E3C;
background-color: #4CAF50; /* 明显的绿色 */
color: #fff;
font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>脱机</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_xiangjichufa">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: red;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
}

QPushButton:pressed {
    border-color: #388E3C;
background-color: #4CAF50; /* 明显的绿色 */
color: #fff;
font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>相机触发</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_shoudongchongkong">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: red;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
}

QPushButton:pressed {
    border-color: #388E3C;
background-color: #4CAF50; /* 明显的绿色 */
color: #fff;
font-weight: bold;
}</string>
          </property>
          <property name="text">
           <string>手动冲孔</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
